### ISSUE TITLE
Scoped concepts

### ISSUE THREAD

**Issue by wfdd** [OpenGovLD/skos#6]
It seems useful to apply [spatial] scope to some of these concepts, like _Hauptausschuss_, which I understand to be specific to Germany and Austria. Presumably, this would be done using `skos:scopeNote`.

**Comment by akuckartz** [comment]
Yes, this is an important topic. I would like such information to be as machine readable as possible. (Re-)using a property based on ISO country codes should be ok.

(In Germany _Hauptausschuss_ is covered by _state_ laws ("Gemeindeordnung"), not federal law. A _Hauptausschuss_ in a municipality in one German state could be considered to belong to a different concept than a _Hauptausschuss_ in another municipality in a different German state. But for the level of abstraction I have in mind it makes sense to use one concept for both.)
